functions/main.go

```diff
@@ -81,6 +81,7 @@ var Functions = []struct {
 	{"Unique", "unique.go", ForNumbersAndStrings, "n"},
 	{"Unshift", "unshift.go", ForAll, "n"},
 	{"Values", "values.go", ForMaps, "n"},
+	{"Remove", "remove.go", ForNumbersAndStrings, "n⋅n"},
```

Comment:
Please keep these lines sorted by name.

Also, O(n^2) isn't quite correct, it would be O(kn) but more on that below.